en/locale.en.json

```diff
@@ -0,0 +1,48 @@
+        "CJ.Lobby.ToHub.Body": "Join other players through this door.\n\nCome on in and enjoy the event!",
+        "CJ.Lobby.ThemeReveal.Title": "Today's theme:",
+        "CJ.Lobby.ThemeReveal.Body": "<b>Creator Jam 282 - Echolocation Jam</b>\r\n\r\nSilent Spring - Create based on an echolocation mechanic.",
+        "CJ.Lobby.ElevatorWelcome.ReturningBody": "Looks like you've been here before.\n\nWhere would you like to go?",
+        "CJ.Lobby.ElevatorWelcome.ReturningTitle": "Welcome back!",
+        "CJ.Lobby.ElevatorWelcome.ReturningStay": "<b>Stay in Lobby</b>",
+        "CJ.Lobby.Team.Title": "Creator Jam Team",
+        "CJ.Lobby.Team.Body": "Creator Jam's team of volunteers is an international diverse group that helps make all of the events run smoothly.",
+        "CJ.Lobby.Directors.Title": "Board of Directors",
+        "CJ.Lobby.Directors.Body": "Creator Jam's Board of Directors is an international diverse group dedicated to improving the organization with their specific expertise.",
+        "CJ.Lobby.MissionStatement.Body": "Creator Jam is a nonprofit fostering creativity and community in the Metaverse. We empower creators through events, resources, and a supportive network.",
+        "CJ.Lobby.MissionStatement.Title": "A world where creators thrive!",
+        "CJ.BuildHub.Guidelines.Title": "Guidelines",
+        "CJ.BuildHub.Guidelines.Building.Header": "<b>Building:</b>",
+        "CJ.BuildHub.Guidelines.Building.Body": "Anyone is welcome to build. No skill is necessary.\r\nCreation is encouraged.\r\nIf you want to create, open the CJ Help menu and select:",
+        "CJ.BuildHub.Guidelines.Building.Subheading": "\"I want to build!\"",
+        "CJ.BuildHub.Guidelines.Licensing.Header": "<b>Licensing:</b>",
+        "CJ.BuildHub.Guidelines.Licensing.Body": "Credit yourself. People want to know what you've made!\r\n\r\nUse <i>CC-0(Public Domain)</i> or <i>CC-BY(Attribution)</i> license assets.\r\n\r\nAll Creations in Creator Jam will be licensed at the end of the jam as:",
+        "CJ.BuildHub.Guidelines.Licensing.Subheading": "<i>Creative Commons CC-BY 4.0 license (Attribution)</i>",
+        "CJ.BuildHub.Guidelines.Precautions.Header": "<b>Precautions:</b>",
+        "CJ.BuildHub.Guidelines.Precautions.Body": "Use caution when importing assets. Assets with a large polygon count can cause performance issues.\r\n\r\n1-5000 polygon count is recommended.\r\n\r\n5000-50,000 polygon count is ok but use caution when using many assets with larger polygon counts.\r\n\r\nLarge file size may cause slow loading times for the user and possibly cause performance\r\nissues.",
+        "CJ.BuildHub.Guidelines.Misc.Header": "Miscellaneous:",
+        "CJ.BuildHub.Guidelines.Misc.Body": "Save your work in the inventory often.\r\n\r\nClean up any mess you make.\r\n\r\nAsk for permission when interacting with someone's unfinished work.\r\nDo not clean up other people’s areas unless specifically asked.\r\n<br><br>"
```

Comment:
This should be "messes" instead of "mess".

> Clean up any messes you make.

Reply:
Corrected. Thank you!